src/js/angular/clustermanagement/templates/cluster-list.html

```diff
@@ -57,7 +57,7 @@
                                 gdb-tooltip="{{'cluster_management.update_cluster_group_dialog.actions.replace_node' | translate}}">
                             <i class="fa-regular fa-arrow-right-arrow-left"></i>
                         </button>
-                        <button ng-click="deleteNode($index)" class="btn btn-link delete-node-btn secondary"
+                        <button ng-click="deleteNode($index, node)" ng-disabled="clusterView.length < 3" class="btn btn-link delete-node-btn secondary"
```

Comment:
Is it possible this to be some property in the model and to be re/calculated when the model is changed?

Reply:
done